run_local_outline.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+echo "Running end2end test suite locally"
+./node_modules/.bin/wdio --suite secnariooutline --baseUrl https://dev-register-a-food-business.azurewebsites.net/new/ wdio.conf.js
```

Comment:
I think this should include the ${URL_ENV} variable like the other files? So it defaults to the test environment but can be run against any environment given that URL_ENV is provided?

We do however just have the e2e tests running against the test env, so it's up to you depending on what the test needs to do.